pom.xml

```diff
@@ -158,6 +173,12 @@
                 <version>${google.oauth.client.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.symphony</groupId>
+                <artifactId>crypto</artifactId>
+                <version>${symphony.crypto.version}</version>
```

Comment:
Once this dependency is not open-sourced it will break Travis CI build. Then, our PR checker will be unavailable.

I think we need to discuss with all team members about this. I mean we need to think another approach to make sure our PR's will be checked.